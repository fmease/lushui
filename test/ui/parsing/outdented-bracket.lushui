# @Task better error message

main = lengthy-space-filler (case 0 of
    \\n => n
)

