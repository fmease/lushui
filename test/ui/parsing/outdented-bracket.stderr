error[E010]: found line break but expected ‘::’, ‘'’, argument, ‘,’, ‘)’, ‘)’ or expression
  ┌─ ${TEST_FOLDER}/parsing/outdented-bracket.lushui:3:30
  │
3 │   main = lengthy-space-filler (case 0 of
  │ ┌──────────────────────────────┘
4 │ │     \\n => n
)
  │ └────────────┘ while parsing this expression starting here
  │
  ├─ ${TEST_FOLDER}/parsing/outdented-bracket.lushui:4:14
  │
4 │     \\n => n
)
  │            ⟫⟪ unexpected token

error: aborting due to previous error

